sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkTracerMetrics.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.trace;
+
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.LongUpDownCounter;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.api.metrics.MeterProvider;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.sdk.trace.samplers.SamplingDecision;
+
+/**
+ * SDK metrics exported for started and ended spans as defined in the <a
+ * href="https://opentelemetry.io/docs/specs/semconv/otel/sdk-metrics/#span-metrics">semantic
+ * conventions</a>.
+ */
+final class SdkTracerMetrics {
+
+  // Visible for testing
+  static final AttributeKey<String> OTEL_SPAN_PARENT_ORIGIN = stringKey("otel.span.parent.origin");
+  // Visible for testing
+  static final AttributeKey<String> OTEL_SPAN_SAMPLING_RESULT =
```

Comment:
You should probably move those into [SemConvAttributes](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/common/src/main/java/io/opentelemetry/sdk/internal/SemConvAttributes.java) with the corresponding tests for up-to-date-ness.